src/components/ResultTextContainer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styled from "styled-components";
+
+const ResultField = styled.div`
+  display: flex;
+  width: 330px;
+  height: 121px;
+  border-radius: 15px;
+  border: #707070 solid 1px;
+  background-color: #c5d5e4;
+`;
+
+export default function ResultTextContainer() {
+  return <ResultField />;
+}
```

Comment:
This component doesn't need to be a React component, you can directly export the `ResultField` and leave these lines out